js/base.js

```diff
@@ -219,6 +219,19 @@ std.daysInMonth = function(month, year) {
     return new Date(year, month, 0).getDate();
 }
 
+document.getElementById('form-autofill-employee-signature').onchange = function (event) {
+    console.info("Reading employee signature")
+    let files = event.target.files
+    // FileReader support
+    if (FileReader && files && files.length) {
```

Comment:
This is bad user experience.
They will not know why it failed.
Instead, I suggest that you check for FileReader availability during loading time and disable the signature field with a message if FileReader is not available.